ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -174,6 +182,8 @@ QObject {
                 root.approvalPending = false
                 root.approvalSuccessful = status == "Success" // TODO: make a all tx statuses Constants (success, pending, failed)
                 d.txHash = ""
+
+                root.swapStore.reevaluateRouterPath(d.uuid, "Paraswap", root.swapFormData.selectedNetworkChainId, true)
```

Comment:
Is this always to be hardcoded to "Paraswap"?

Reply:
At least for now, cause atm "Paraswap" is the only swap provider we support.

Reply:
Ok, then I'd just replace it with `Constants.swap.paraswapName`, wdyt?

Reply:
Not a problem, I made that function generic, cause we will need it for the bridge as well. 
What I will do is create a new one like `reevaluareSwap`  on the qml side, but the module's logic will take care and add `Paraswap ` when calls `reevaluateRouterPath`.